.jvmopts

```diff
@@ -1,4 +1,3 @@
 -Xss16M
 -Xmx4G
 -XX:+UseCompressedOops
---upgrade-module-path=engine/runtime/build-cache/truffle-api.jar
```

Comment:
After all these years, we can finally get rid of the workaround [introduced in ancient times](https://github.com/enso-org/enso/pull/928) (3 years ago). Hurray! 🎉